c_src/erlang_jq_port.c

```diff
@@ -195,8 +195,8 @@ static byte* read_packet() {
     }
     len = 0;
     for (int i = (PACKET_SIZE_LEN-1); i >= 0; i--) {
-        size_t addad_byte = ((size_t)buf[PACKET_SIZE_LEN - (i+1)] << (i*sizeof(byte)));
-        len = len | addad_byte;
+        size_t addad_byte = (size_t)(buf[PACKET_SIZE_LEN - (i+1)]);
+        len = len | (addad_byte << (i * 8));
```

Comment:
NIT,

maybe 
```
#include <limits.h>
....
```

and then
use CHAR_BIT
```
len = len | (addad_byte << (CHAR_BIT*sizeof(byte) * i));
```

Reply:
Thank you, I did not know about this. Will fix it in a later commit.